R/code7.r

```diff
@@ -39,10 +39,11 @@ print(head(imputeOut))
 
 # ---- code7-b ----
 # Read in file containing protein coding genes coords
-genes <- read.csv(protein.coding.coords.fname)
+genes <- read.csv(protein.coding.coords.fname, stringsAsFactors = FALSE)
 
 # Subset for CETP SNPs
 impCETP <- map2gene("CETP", coords = genes, SNPs = imputeOut)
+impCETP$SNP <- as.character(impCETP$SNP)
```

Comment:
This is odd.  What is the class of impCETP$SNP?  In map2gene, the return value uses the stringsAsFactors=FALSE option, so SNP should be character already.